src/test/features/claimant-response/routes/choose-how-to-proceed.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+import { checkAuthorizationGuards } from 'test/features/claim/routes/checks/authorization-check'
+
+import { Paths } from 'claimant-response/paths'
+
+import { app } from 'main/app'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as draftStoreServiceMock from 'test/http-mocks/draft-store'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+
+import { FormaliseRepaymentPlanOption } from 'features/claimant-response/form/models/formaliseRepaymentPlanOption'
+
+const cookieName: string = config.get<string>('session.cookieName')
+const externalId = claimStoreServiceMock.sampleClaimObj.externalId
+const pagePath: string = Paths.chooseHowToProceedPage.evaluateUri({ externalId: externalId })
+const pageContent: string = 'Choose how to proceed'
+const defendantPartialAdmissionResponse = claimStoreServiceMock.sampleDefendantPartialAdmissionResponseObj
+
+describe('Claimant response: choose how to proceed page', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+
+    checkAuthorizationGuards(app, 'get', pagePath)
+
+    describe('for authorized user', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      })
+
```

Comment:
Please add tests simulating error in draft retrieval.

Reply:
done